doc/md/fundamentals/1-basic-syntax/1-defining-an-actor.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+# Defining an actor
+
+In Motoko, an actor is a unit of computation that encapsulates [state](https://internetcomputer.org/docs/motoko/fundamentals/state) and behavior. Unlike traditional functions or objects in other programming languages, actors operate independently and interact with each other through [asynchronous](https://internetcomputer.org/docs/motoko/fundamentals/actors-async#async--await) messaging.
+
+Actors are objects with special system-level capabilities. An actor supports asynchronous operations and can persist state across calls. Each actor runs independently with its own message queue, allowing concurrent execution.  
+
+A Motoko program typically starts by defining an actor.
+
+```motoko
```

Comment:
Tbh, although boring, I think the counter is actually a better example since it has some state. Then you can add an increment update method and a read() query method. This illustrated encapuslation too as the counter value can only increase by calling inc(), never decrease, and the count is only accessible via inc() and read() and cannot be modified otherwise.